Robot Code/src/org/usfirst/frc/team3316/robot/commands/chassis/ToggleOmni.java

```diff
@@ -0,0 +1,47 @@
+package org.usfirst.frc.team3316.robot.commands.chassis;
+
+import org.usfirst.frc.team3316.robot.commands.DBugCommand;
+
+public class ToggleOmni extends DBugCommand
+{
+	private static boolean set = true;
+	
+	private DBugCommand extend, retract;
+	
+	public ToggleOmni ()
```

Comment:
Most likely not, I think it will cause too much hassle for something that the driver can easily solve.